.gitignore

```diff
@@ -30,5 +30,3 @@ src/main/docs/resources/css/*.css
 src/main/docs/resources/js/*.js
 src/main/docs/resources/style/*.html
 src/main/docs/resources/img/micronaut-logo-white.svg
-*Generated*.java
-/aws-ua/src/main/java/io/micronaut/aws/ua/GeneratedUserAgentProvider.java
```

Comment:
@sdelamo This file is still generated by the build, and ends up being an untracked file?

Reply:
Actually, this is fixed by https://github.com/micronaut-projects/micronaut-aws/pull/1619/commits/31c8caaf2f9d36e0ccfd35fc7ad6bcc592d8d980